src/components/FrameworkLabel.astro

```diff
@@ -3,9 +3,11 @@ import FRAMEWORKS from '../frameworks';
 const {id, size = 20} = Astro.props
 
 const framework = FRAMEWORKS.find((f) => f.id === id);
+
+const baseURL = import.meta.env.MODE === 'development' ? '' : 'https://raw.githubusercontent.com/matschik/component-party/main/public/';
```

Comment:
Wow nice !! You solved the image problem !! 💯

Reply:
I'm not nearly as familiar with Astro as you are nor am I familiar with how you deploy, but as long as you use the production mode in deployment I think it should work!